maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Helpers/ScreenSizeHelper.swift

```diff
@@ -0,0 +1,47 @@
+import XCTest
+
+struct ScreenSizeHelper {
+    static func physicalScreenSize() -> (Float, Float) {
+        let springboardBundleId = "com.apple.springboard"
+        let springboardApp = XCUIApplication(bundleIdentifier: springboardBundleId)
+        let screenSize = springboardApp.frame.size
+        return (Float(screenSize.width), Float(screenSize.height))
+    }
+
+    private static func actualOrientation() -> UIDeviceOrientation {
+        let orientation = XCUIDevice.shared.orientation
+        if orientation == .unknown {
+            // If orientation is "unknown", we assume it is "portrait" to
+            // work around https://stackoverflow.com/q/78932288/7009800
+            return UIDeviceOrientation.portrait
```

Comment:
what would be the implications of setting it to portrait as a side-effect here, when it's unknown?

Reply:
As of now, I'm not aware of any. See ["good things"](https://github.com/mobile-dev-inc/maestro/pull/1974#issuecomment-2337502676) above.